whirlpool/auth.py

```diff
@@ -58,49 +59,42 @@ def _save_auth_data(self):
         with open(AUTH_JSON_FILE, "w") as f:
             json.dump(self._auth_dict, f)
 
-    async def _do_auth(self, refresh_token: str) -> str | None:
-        auth_url = f"{self._backend_selector.base_url}/oauth/token"
-        auth_header = {
-            "Content-Type": "application/x-www-form-urlencoded",
-            # "Brand": "Whirlpool",
-            # "WP-CLIENT-REGION": "EMEA",
-            # "WP-CLIENT-BRAND": "WHIRLPOOL",
-            # "WP-CLIENT-COUNTRY": "EN",
-        }
-
-        auth_data = {
-            "client_id": self._backend_selector.client_id,
-            "client_secret": self._backend_selector.client_secret,
-        }
-
+    def _get_auth_body(
+        self, refresh_token: str, creds: Dict[str, str]
+    ) -> Dict[str, str]:
         if refresh_token:
             LOGGER.info("Fetching auth with refresh token")
```

Comment:
```suggestion
            LOGGER.info("Using refresh token in auth body")
```
Since this method is not the one doing the fetch anymore, lets update the log too :)